start.sh

```diff
@@ -0,0 +1,3 @@
+sleep 10
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345754)